src/test/java/hudson/remoting/ChannelRunners.java

```diff
@@ -0,0 +1,21 @@
+package hudson.remoting;
+
+import java.util.stream.Stream;
+
+public final class ChannelRunners {
+    public static final String PROVIDER_METHOD = "hudson.remoting.ChannelRunners#provider";
+
+    private ChannelRunners() {}
+
+    @SuppressWarnings("unused") // used by JUnit
+    static Stream<ChannelRunner> provider() {
```

Comment:
```suggestion
    public static Stream<ChannelRunner> provider() {
```
? Seems weird to be accessing a non`public` method reflectively.